### PR TITLE
linkerd 2.10 announcement blog post

### PR DESCRIPTION
**Pull request by wmorgan**
Signed-off-by: William Morgan <william@buoyant.io>

### CODE REVIEW

File: linkerd.io/content/blog/announcing-linkerd-2-10.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+Multi-cluster support, [introduced in Linkerd
+2.8](https://linkerd.io/2020/06/09/announcing-linkerd-2.8/), allows Linkerd to
+connect Kubernetes services across cluster boundaries in a way that's secure,
+fully transparent to the application, and independent of the topology of the
+underlying network. However, this functionality was restricted to HTTP
+connections only—until now. With Linkerd 2.10, Linkerd's multi-cluster feature
+now extends to all TCP connections, with the same guarantees of security and
+transparency that Linkerd provides for pod-to-pod communication.
+
+Want to try it? Just install the `multicluster` extension!
```
Comment (dadjeibaah) [inline linkerd.io/content/blog/announcing-linkerd-2-10.md:88]: Should we add a link to our multicluster docs here for folks to try out multicluster?

File: linkerd.io/content/blog/announcing-linkerd-2-10.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+The service mesh doesn't have to be complex, and security doesn't have to be
+high-friction. The future of Linkerd is built around these beliefs, and we hope
+they resonate with you as well.
+
+## Try it today!
+
+Ready to try Linkerd? Those of you who have been tracking the 2.x branch via
+our [weekly edge releases](https://linkerd.io/2/edge) will already have seen
+these features in action. Either way, you can download the stable 2.9 release
```
Comment (dadjeibaah) [inline linkerd.io/content/blog/announcing-linkerd-2-10.md:162]:
```suggestion
these features in action. Either way, you can download the stable 2.10 release
```

**Review by dadjeibaah — COMMENTED** [review]
Some minor comments but this all looks great!

File: linkerd.io/content/blog/announcing-linkerd-2-10.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+bringing _policy_ to Linkerd, building on the foundation of mTLS to further
+enhance the security posture of Kubernetes applications everywhere.
+
+The service mesh doesn't have to be complex, and security doesn't have to be
+high-friction. The future of Linkerd is built around these beliefs, and we hope
+they resonate with you as well.
+
+## Try it today!
+
+Ready to try Linkerd? Those of you who have been tracking the 2.x branch via
```
Comment (adleong) [inline linkerd.io/content/blog/announcing-linkerd-2-10.md:160]: what's 2.x?  Should this be 2.10?

**Pull request merged**